tests/integration/conftest.py

```diff
@@ -27,36 +29,63 @@
 @pytest_asyncio.fixture(name="deploy", scope="module")
 async def deploy(ops_test: OpsTest):
```

Comment:
I think we could parameterise a single deploy fixture instead of two. Maybe something like https://stackoverflow.com/questions/18011902/how-to-pass-a-parameter-to-a-fixture-function-in-pytest
What do you think?

Reply:
I agree. Let me check.

Reply:
I checked it, but it's not that nice when we do it. For example, the fixture and the tests should change like this:


```python3
# Tests
@pytest.mark.parametrize("deploy", [False], indirect=["deploy"])
@pytest.mark.usefixtures("deploy")
class TestDeployment:
   # ...
   
# Fixture
@pytest.mark.skip_if_deployed
@pytest_asyncio.fixture(name="deploy", scope="module")
async def deploy(ops_test: OpsTest, request):
    """Build and deploy the app and its components."""
    deploy_current_stable = request.param
    await deploy_package(ops_test, deploy_current_stable)
```

To me, it seems a bit unclear and patchy. What do you think?